### DISCUSSION TITLE
cannot convert a slice of length 20 to FixedBytes<32>

### DISCUSSION THREAD

**Discussion by Akashh-Dutta** [foundry-rs/foundry#7885] [Help]
Hey guys, I am totally new in this field, and I am learning from youtube. and I tried to put private key to deploy in anvil and it is showing this error.
''' Message:  cannot convert a slice of length 20 to FixedBytes<32>
Location: crates/wallets/src/wallet_signer.rs:51 '''

Here is the full error message-
'''
The application panicked (crashed).
Message:  cannot convert a slice of length 20 to FixedBytes<32>
Location: crates/wallets/src/wallet_signer.rs:51

This is a bug. Consider reporting it at https://github.com/foundry-rs/foundry

  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ BACKTRACE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
                                ⋮ 7 frames hidden ⋮                               
   8: foundry_wallets::wallet_signer::PendingSigner::unlock::he6f92e37830eb2b9
      at <unknown source file>:<unknown line>
   9: foundry_wallets::raw_wallet::RawWalletOpts::signer::h5854bbee03e9b0b3
      at <unknown source file>:<unknown line>
  10: forge::cmd::create::CreateArgs::run::{{closure}}::h4d3bf6d966dc4613
      at <unknown source file>:<unknown line>
  11: forge::main::hc8bf1a9e91f73903
      at <unknown source file>:<unknown line>
  12: std::sys_common::backtrace::__rust_begin_short_backtrace::he00772b07c27e46c
      at <unknown source file>:<unknown line>
  13: main<unknown>
      at <unknown source file>:<unknown line>
  14: __libc_start_main<unknown>
      at <unknown source file>:<unknown line>
  15: _start<unknown>
      at <unknown source file>:<unknown line>

Run with COLORBT_SHOW_HIDDEN=1 environment variable to disable frame filtering.
Aborted
'''
thanks for reading.

**Comment by DaniPopes** [comment]
This was fixed in the last version
